### ISSUE TITLE
Issue with 'Modeling Sequential Data Using RNN

### ISSUE THREAD

**Issue by mrinal-sourav** [rasbt/deep-learning-book#78]
So, 
I tried to follow through the RNN example provided through pages 552 to 566. 
There seems to be an issue with dimensions for the logits and labels. 
Attaching the '.pdf' of the notebook. 
I tried adding dimensions to the 'y_train' data but it still doesn't work. 

[RNN_model.pdf](https://github.com/rasbt/deep-learning-book/files/3137827/RNN_model.pdf)

Please let me know where I am wrong. 
Thank you.

**Comment by rasbt** [comment]
Are you referring to the Python Machine Learning Book (2nd edition) Ch. 16?

**Comment by mrinal-sourav** [comment]
My bad. 
Yes. 
Chapter 16. 
I wish I could share the Jupyter notebook or the '.html' but this forum doesn't allow. 
Please let me know if reading the '.pdf' is an issue.

**Comment by rasbt** [comment]
Hm that's weird. There might be a typo somewhere in your code. I just took the original code from the Jupyter Notebook we shared (https://github.com/rasbt/python-machine-learning-book-2nd-edition/blob/master/code/ch16/ch16.ipynb) and ran this on the latest version of TensorFlow (v1.13.1) and it works without any issue. 

I just uploaded a shortened version of that notebook via Google Colab if you want to try: https://colab.research.google.com/drive/1L-ppNmZQLDs394xtdGJl34bj9e1uKeBD. Did you type the code from the book? Maybe an accidental typo happened during this process?
